sbc_compassion/security/ir.model.access.csv

```diff
@@ -8,4 +8,4 @@
 "access_correspondence_type","Read access on correspondence_type","model_sponsorship_correspondence_type","child_compassion.group_sponsorship",1,0,0,0
 "access_correspondence_lang_checkbox","Read access on correspondence_checkbox","model_sponsorship_correspondence_lang_checkbox","child_compassion.group_sponsorship",1,0,0,0
 "access_import_letters_history","Access on import_letters_history","model_import_letters_history","child_compassion.group_sponsorship",1,1,1,0
-"access_import_letters_line","Access on import_letters_line","model_import_letter_line","child_compassion.group_sponsorship",1,1,0,1
+"access_import_letters_line","Access on import_letters_line","model_import_letter_line","child_compassion.group_sponsorship",1,1,1,1
```

Comment:
If I remember correctly, we said that we do not want people creating a letter_line without using the import